test/bats/plugin-test.bats

```diff
@@ -81,8 +81,7 @@ SLEEP_TIME=1
     run kubectl replace -f ./config/samples/config_v1beta1_store_oras.yaml
     sleep 5
 
-    run kubectl run cosign-demo-keyless --namespace default --image=wabbitnetworks.azurecr.io/test/cosign-image:signed-keyless
-    assert_success
+    wait_for_process 30 10 'kubectl run cosign-demo-keyless --namespace default --image=wabbitnetworks.azurecr.io/test/cosign-image:signed-keyless'
```

Comment:
I remember the stuck case would take ~5 minutes, maybe we only need to retry once in case it takes too much time?

Reply:
sure I can reduce to 1. 3 was just arbitrary on my end.